atomic_reactor/utils/remote_host.py

```diff
@@ -23,9 +23,9 @@
 SSH_COMMAND_TIMEOUT = 30
 SLOTS_RELATIVE_PATH = "osbs_slots"
 RETRY_ON_SSH_EXCEPTIONS = (paramiko.ssh_exception.NoValidConnectionsError,
-                           paramiko.ssh_exception.SSHException)
+                           paramiko.ssh_exception.SSHException, ConnectionError, TimeoutError)
 BACKOFF_FACTOR = 3
-MAX_RETRIES = 3
+MAX_RETRIES = 30
```

Comment:
How did you decide on 30? The formula for the backoff seems to be
```
factor * 2 ** n
```
where the factor is currently 3 and n is the number of the try, thus with the current settings this would continue for a **very** long time until the build timeouted. Is that what we want?

Reply:
changed